packages/docs/docs/cli/external-fhir-servers.md

```diff
@@ -19,34 +19,87 @@ To enable these scenarios, you will need a `clientId` or `clientSecret` to acces
 
 The examples below use the [CLI optional flags](/docs/cli#optional-flags).
 
+## Setting up your credentials
+
+Medplum CLI stores credentials to be used in a future period without needing it to be entered in every command. By using the `profile` command, this helps with the ability to work with multiple FHIR servers.
+
+### Example: Setting a Profile
+
+In this example, we will set up a profile using `medplum profile set <profileName>` with the flags below:
+
+- --auth-type
```

Comment:
Agree with @reshmakh  - we should list out possible values for `--auth-type`.  I'm not sure if we should do it here, or put a table below.